scripts/common.py

```diff
@@ -390,6 +390,20 @@ def verify_current_branch_name(expected_branch_name):
             expected_branch_name)
 
 
+def is_port_in_use(port):
+    """Checks if a port is in use or not.
+
+    Args:
+        port: int. The port number.
+
+    Returns:
+        bool. True if port is in use else False.
+    """
+    with contextlib.closing(
+        socket.socket(socket.AF_INET, socket.SOCK_STREAM)) as s:
+        return bool(s.connect_ex(('localhost', port)))
```

Comment:
I think this function should replace `is_port_open` since `is_port_open` is wrong